content/docs/using-regolith/install.md

```diff
@@ -15,6 +15,41 @@ If you're upgrading from an earlier version of Regolith, you may be interested i
 ### Ubuntu
 
 {{< tabs "ubuntu-tabs" >}}
+{{< tab "Ubuntu 24.10" >}}
```

Comment:
In the past it's been somewhat common of an installation issue that people choose the wrong version to install from.  I wonder how much non LTS releases that have been deprecated will be used...  I wonder if it would be better to stick with supported LTS + latest release for tabs on the install page, and a note somewhere explaining which older versions we also support.  Thoughts?

Reply:
I didn't get your point. that eventually will be up to the users to know which versions they are installing. We've been removing deprecated non-LTS instructions from the site too, but the release for 24.10, which is another non-LTS, is going to be the latest release. IMO we should have an instruction for it, and probably remove it at some point in the future when the it's reached EOL (either from upstream, or from Regolith).